pandas/core/frame.py

```diff
@@ -5168,9 +5168,8 @@ def _get_agg_axis(self, axis_num):
 
     def mode(self, axis=0, numeric_only=False):
         """
-        Gets the mode(s) of each element along the axis selected. Empty if
```

Comment:
@jreback — not sure how to do this, I reckon this is to reference the same-ish doc string in all relevant places?